apis/alerts/v1alpha1/kafka_alerts_types.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright AppsCode Inc. and Contributors
+
+Licensed under the AppsCode Community License 1.0.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://github.com/appscode/licenses/raw/1.0.0/AppsCode-Community-1.0.0.md
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	mona "kmodules.xyz/monitoring-agent-api/api/v1"
+	api "x-helm.dev/apimachinery/apis/releases/v1alpha1"
+)
+
+const (
+	ResourceKindKafkaAlerts = "KafkaAlerts"
+	ResourceKafkaAlerts     = "Kafkaalerts"
+	ResourceKafkaAlertss    = "kafkaalertss"
+)
+
+// MongodbAlerts defines the schama for KubeDB Ops Manager Operator Installer.
```

Comment:
there are a bunch of `mongodb` inside the `kafka-alerts` folder also. Change them all.

Reply:
https://github.com/appscode/alerts/actions/runs/8294818988/job/22700681927?pr=57#step:6:150

Remove `successfulJobsHistoryLimit` field

Reply:
removed